src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -52,6 +52,12 @@
 (def SemVerVersion
   (schema/pred semver/valid-format? "semver"))
 
+(def StatusProxyConfig
+  {:proxy-target-url schema/Str
```

Comment:
I decided to go with the stand alone validation function, because when I used `schema/pred` the error it gave wasn't very clear about what the actual problem was. Maybe there's a way to put a custom error message when using a predicate though?